app/config.hsl.js

```diff
@@ -1,6 +1,6 @@
 const CONFIG = process.env.CONFIG || 'hsl';
 const API_URL = process.env.API_URL || 'https://dev-api.digitransit.fi';
-const APP_DESCRIPTION = 'HSL:n uuden Reittioppaan kokeiluversio.';
+const APP_DESCRIPTION = 'Helsingin seudun liikenteen uusi Reittiopas.';
```

Comment:
From slack discussion. It's obviously time to remove the "kokeiluversio" and this just essentially says the same.